podium/rivulet
# Cody/avro consumers

Get consumption of avro-serialized topics working.

Explanations in comments of files.

QA Steps:
1. Clone, mix.deps.get, mix compile. Reject PR on compile errors.
2. Uncomment pipeline in `application.ex`. Remember to put a comma on the line before.
3. Run `iex -S mix`. Reject PR if crashes.
4. Verify that the contents of your RR users table are successfully pulled off kafka, decoded, and printed. If you see a bunch of kafka message structs containing avro data using erlang data structures, it has succeeded.
